QUARKUS-1070.md

```diff
@@ -0,0 +1,39 @@
+# QUARKUS-1070 Composable Platform streams and non-platform extension discovery
+
+Jira: https://issues.redhat.com/browse/QUARKUS-1070
+
+The goal of this epic is to
+
+* introduce a registry.quarkus.redhat.com for all devtools to consult for looking up platform and extensions
+* introduce bom generator in platform that generates uniform but composable BOMs that can be imported individually 
+  rather than all in bulk 
+* introduce notion of platform stream to indicate which minor version you are tracking and/or what you want to target 
+  when using tools
+
+## Scope of the testing
+
+### Impact on test suites and testing automation
+* Red Hat build of Quarkus platform configuration has to be included in Maven repository tests
+* Quickstarts with RHBQ supported extensions have to be buildable and runnable with BOMs included in the Red Hat build 
+  of Quarkus platform releases
+* Code starter tests in start-stop test suite have to pass with code.quarkus portal updated with Red Hat build of 
+  Quarkus platform releases
+
```

Comment:
* Mandrel version considerations:
   * Initial platform release (RHBQ) needs to be tested against the internal build of Mandrel as Mandrel is usually released hand in hand with RHBQ
   * SP releases of the platform need to be tested with the public version of Mandrel as the internal registry can move ahead